Add note on duplicate items' relationships (#150)
directus/docs: content/guides/1.data-model/1.collections.md

The **Save as Copy** option in :product-link{product="editor"} offers a way to effectively duplicate the current item. Duplication settings define which field values will be copied.

::callout{icon="material-symbols:info-outline"}

**Duplicating Items with Relationships**<br/>

> Note that when duplicating an item, the relationships need to be set once again on the duplicate item.

Reviewer: To me this reads quite ambiguously.

> Note that when duplicating an item, the relationships need to be set once again on the duplicate item.

Author: ```suggestion
When you duplicate an item, its relationships are not copied. You must redefine these relationships for the new item.
```

> When you duplicate an item, its relationships are not copied. You must redefine these relationships for the new item.

Reviewer: ```suggestion
When you duplicate an item, any related items are not copied. You must create new relationships between the duplicated item and related items.
```

Reviewer: "Relationships" is a hard word because it refers to the data modeling. But here it means related items.

Author: This is perfect, thank you!
